### PR TITLE
Update the develop branch

### PR DESCRIPTION
**Pull request by cservan**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+set -eou pipefail
+
+echo "Prefix set to $PREFIX"
+export CMAKE_PREFIX_PATH=$PREFIX
+
+export CXXFLAGS="-I$PREFIX/include -L$PREFIX/lib"
+
+set -eou pipefail
```
Comment (maudetes) [inline install.sh]: `set -eou pipefail` appears twice

File: test/main_NLP.cc [hunk 2]
```diff
@@ -212,18 +221,40 @@ int main ( int argc, char *argv[] )
...
+            l_output_vec = stemming(l_output_vec,stem);
+        if (l_stopwords) 
+            l_output_vec = filtering_stopwords(l_output_vec,sw);
+        if (l_generalize) 
+            l_output_vec  = generalize(l_output_vec,l_generalize);
         if ((int)l_BPE.size() != 0)
         {
+            if (! l_spm_model)
+            {
+//                BPE = new bpemodel(l_BPE);
```
Comment (maudetes) [inline test/main_NLP.cc]: Clean commented lines

File: README.md [hunk 3]
```diff
@@ -2,9 +2,20 @@
...
+>&nbsp;&nbsp;&nbsp;&nbsp;-p PREFIX&nbsp;&nbsp;&nbsp;&nbsp;specify a prefix (default /usr/local/) <br>
+
+
+
 # Tools available
-## qtokenizer
+## qnlp-tokenizer
 The very first step for every NLP process is a tokenizer: a tool which segement text into tokens. These tokens  can be words or set of words. For example, "c'est-à-dire" or "aujourd'hui" are a tokens. The tokenizations rules are available for every european languages, but specific rules have been written for French and English.
+## qnlp-filtering
+Enable to filter bitext according the sentence length, the sentence length ratio and other tricks.
```
[the remark this answers is not in the document]
Comment (cservan) [reply README.md]: Add additionnal information.

File: install.sh [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        ;;
+    g)  NOT_USE_BUILTIN_PROTOBUF=1
+        ;;
+    p)  PREFIX=$OPTARG
+        ;;
+    esac
+done
+
+
+#if [ -n "$1" ]
```
[the remark this answers is not in the document]
Comment (cservan) [reply install.sh]: Ok, done.

File: CMakeLists.txt [hunk 5]
```diff
@@ -1,91 +1,56 @@
...
-# set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_BUILD_TYPE Release)
-SET(CMAKE_VERBOSE_MAKEFILE ON)
-set(CMAKE_CXX_FLAGS  " ${CMAKE_CXX_FLAGS} -std=c++11 -fPIC -O3 -Ofast -m64 -march=native -funroll-loops -ffinite-math-only -Wno-unused-result -Wno-deprecated -pthread -g -fpermissive")
-# set(CMAKE_CXX_FLAGS " -O2 -std=c++11 -lpthread -fpermissive ")
-MESSAGE( STATUS "CMAKE_CXX_FLAGS:         " ${CMAKE_CXX_FLAGS})
-find_package(OpenMP)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-MESSAGE( STATUS "CMAKE_CXX_FLAGS:         " ${CMAKE_CXX_FLAGS})
+# add_subdirectory(vendor/fastText)
```
[the remark this answers is not in the document]
Comment (cservan) [reply CMakeLists.txt]: Ok, done.

File: test/main_NLP.cc (hunk 2 above)
Comment (cservan) [reply test/main_NLP.cc]: Ok, done.

**Pull request closed without merging**